examples/1d_stencil/1d_stencil_1.cpp

```diff
@@ -105,32 +106,32 @@ int hpx_main(hpx::program_options::variables_map& vm)
     std::uint64_t const os_thread_count = hpx::get_os_thread_count();
     print_time_results(os_thread_count, elapsed, nx, nt, header);
 
-    return hpx::finalize();
+    return hpx::local::finalize();
 }
 
 int main(int argc, char* argv[])
 {
     namespace po = hpx::program_options;
 
+    // clang-format off
     po::options_description desc_commandline;
-    desc_commandline.add_options()
-        ("results", "print generated results (default: false)")
-        ("nx", po::value<std::uint64_t>()->default_value(100),
-         "Local x dimension")
-        ("nt", po::value<std::uint64_t>()->default_value(45),
-         "Number of time steps")
-        ("k", po::value<double>(&k)->default_value(0.5),
-         "Heat transfer coefficient (default: 0.5)")
-        ("dt", po::value<double>(&dt)->default_value(1.0),
-         "Timestep unit (default: 1.0[s])")
-        ("dx", po::value<double>(&dx)->default_value(1.0),
-         "Local x dimension")
-        ( "no-header", "do not print out the csv header row")
-    ;
+    desc_commandline.add_options()(
+        "results", "print generated results (default: false)")("nx",
+        po::value<std::uint64_t>()->default_value(100),
+        "Local x dimension") ("nt",
+        po::value<std::uint64_t>()->default_value(45),
+        "Number of time steps") ("k", po::value<double>(&k)->default_value(0.5),
+        "Heat transfer coefficient (default: 0.5)") ("dt",
+        po::value<double>(&dt)->default_value(1.0),
+        "Timestep unit (default: 1.0[s])") ("dx",
+        po::value<double>(&dx)->default_value(1.0),
+        "Local x dimension") ("no-header",
+        "do not print out the csv header row");
+    // clang-format on
```

Comment:
Can we avoid reformatting the program options, please?

Reply:
Yes. I've tried to them from clang-formatting but missed this one. Thanks!